src/js/providers/tracks-mixin.ts

```diff
@@ -341,7 +342,9 @@ const Tracks: TracksMixin = {
                         continue;
                     }
 
-                    track.mode = mode;
+                    if (mode !== 'disabled' || trackId.indexOf('nativecaptions') !== 0) {
```

Comment:
can i get a quick explainer on why we don't set if the mode is `disabled` for non-native use cases?

Reply:
we can't detect cues in disabled tracks, and we don't consider disabled tracks items in the captions menu.